docs/developer-docs/defi/overview.mdx

```diff
@@ -4,54 +4,137 @@ keywords: [intermediate, defi, overview, tokens, icrc-1, icrc ledger, icrc index
 
 import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
 
-# Overview
+# Digital assets overview
 
-<MarkdownChipRow labels={["Intermediate"  ]} />
+<MarkdownChipRow labels={["Intermediate", "Concept"]} />
+
+A digital asset is an electronically-stored item that has value and can be owned, sold, traded, or transferred.
+
+Digital assets can take many forms, such as:
+
+- Tokens and stablecoins.
+
+- Non-fungible tokens (NFTs).
+
+- Digital records corresponding to physical items, such as real estate or luxury goods.
+
+On ICP, digital assets come in three primary forms: ICP tokens, ICRC tokens, and chain-key tokens.
+
+ICP tokens are the network's native utility token. They are used for different actions on the network, such as being converted into [cycles](/docs/current/developer-docs/getting-started/cycles/overview) to pay for a canister's resources, or being staked in a neuron to receive voting power on the [NNS](/docs/current/developer-docs/daos/nns/overview).
+
+[ICRC](https://github.com/dfinity/ICRC) stands for `Internet Computer Request for Comments` and is a working group for various topics. ICRC standards can be created for anything, not just token types. Currently, the protocol supports the ICRC token standards ICRC-1, ICRC-2, ICRC-3, and ICRC-7, with several more in the process of being drafted and developed.
+
+[Chain-key](/docs/current/developer-docs/multi-chain/chain-key-tokens/overview) tokens are created using ICP's chain-key cryptography protocols, such as [threshold signing](https://internetcomputer.org/docs/current/concepts/glossary/#chain-key). The chain-key token ckBTC uses the ICRC-1 standard, while ckETH and ckERC20 use the ICRC-2 standard. Each chain-key token is deployed on ICP and backed 1:1 with the native token, such as BTC, ETH, or ERC20, held 100% on-chain in an ICP smart contract. There are no intermediaries or centralized bridges used for chain-key tokens.
+
+The older token standards, [DIP-20](https://github.com/Psychedelic/DIP20) and [DIP-721](https://github.com/Psychedelic/DIP721), are designed to resemble the Ethereum ERC-20 and ERC-721 standards respectively. While some projects on ICP still use DIP-20 and DIP-721, the new ICRC-2 and ICRC-7 standards are designed to provide a newer, alternative standard for each.
+
+Each token standard is elaborated on in the [token standards documentation](/docs/current/developer-docs/defi/tokens/token-standards).
+
+## Terminology
```

Comment:
This section reads more like a reference rather then an overview. Ideally all these concepts would be introduced in a text that flows and have diagrams.

Reply:
This terminology definition format was requested specifically by the Identity and Wallet Standards working group.